packages/devtools_shared/lib/src/memory/heap_sample.dart

```diff
@@ -48,37 +48,37 @@ class HeapSample {
   /// Version of HeapSample JSON payload.
   static const version = 1;
 
-  final int timestamp;
+  final int? timestamp;
```

Comment:
The default constructor arg should all be required, the ? isn't necessary?

Reply:
removed